vite.config.ts

```diff
@@ -0,0 +1,13 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+import svgr from "vite-plugin-svgr";
+
+
+// https://vite.dev/config/
+export default defineConfig({
+  plugins: [react(), svgr(),],
+  build: {
+      // This matches Vercel's expectations.
+      outDir: "build",
```

Comment:
Vite defaults to putting things in `dist`, so we modify that here so that we don't have to muck with Vercel settings.